### PR TITLE
ADD: Transform function CF1 to CF2 and vice versa

### PR DESCRIPTION
**Pull request by syedhamidali**
- [x] Closes #136  
- [x] Tests added
- [x] Changes are documented in `history.md`

### CODE REVIEW

File: examples/notebooks/Transform.ipynb [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    "file = DATASETS.fetch(\"cfrad.20080604_002217_000_SPOL_v36_SUR.nc\")\n",
+    "dtree = xd.io.open_cfradial1_datatree(file)\n",
+    "dtree"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Transform CF1 to CF2"
```
Comment (kmuehlbauer) [inline examples/notebooks/Transform.ipynb:50]: Isn't this the other way round?

File: examples/notebooks/Transform.ipynb [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file = DATASETS.fetch(\"cfrad.20080604_002217_000_SPOL_v36_SUR.nc\")\n",
+    "dtree = xd.io.open_cfradial1_datatree(file)\n",
```
Comment (kmuehlbauer) [inline examples/notebooks/Transform.ipynb:42]: You might read it as pure netcdf (xr.open_dataset) and the transform to CF2 and back to CF1. But not necessary.

**Review by kmuehlbauer — APPROVED** [review]
LGTM, Slight inconsistency in the notebook. But after that we can merge, I think.

File: examples/notebooks/Transform.ipynb (hunk 1 above)
Comment (syedhamidali) [reply examples/notebooks/Transform.ipynb]: You're right :)

**Comment by kmuehlbauer** [comment]
@syedhamidali Thanks for coming back to this. Great work!

**Comment by syedhamidali** [comment]
You’re my inspiration, @kmuehlbauer. None of this would have been possible without your support every step of the way.

**Pull request merged**